lib/iris/tests/test_pandas.py

```diff
@@ -42,10 +43,306 @@
+        self.assertArrayEqual(series, cube.data)
+        self.assertArrayEqual(series.index, expected_index)
+
+    def test_simple(self):
+        cube = Cube(np.array([0, 1, 2, 3, 4.4]), long_name="foo")
+        dim_coord = DimCoord([5, 6, 7, 8, 9], long_name="bar")
+        cube.add_dim_coord(dim_coord, 0)
+        expected_index = np.array([5, 6, 7, 8, 9])
+        series = iris.pandas.as_series(cube)
+        self.assertArrayEqual(series, cube.data)
+        self.assertArrayEqual(series.index, expected_index)
+
+    def test_masked(self):
+        data = np.ma.MaskedArray([0, 1, 2, 3, 4.4], mask=[0, 1, 0, 1, 0])
+        cube = Cube(data, long_name="foo")
+        series = iris.pandas.as_series(cube)
+        self.assertArrayEqual(series, cube.data.astype("f").filled(np.nan))
+
+    def test_time_standard(self):
+        cube = Cube(np.array([0, 1, 2, 3, 4]), long_name="ts")
+        time_coord = DimCoord(
+            [0, 100.1, 200.2, 300.3, 400.4],
+            long_name="time",
+            units="days since 2000-01-01 00:00",
+        )
+        cube.add_dim_coord(time_coord, 0)
+        expected_index = [
+            datetime.datetime(2000, 1, 1, 0, 0),
+            datetime.datetime(2000, 4, 10, 2, 24),
+            datetime.datetime(2000, 7, 19, 4, 48),
+            datetime.datetime(2000, 10, 27, 7, 12),
+            datetime.datetime(2001, 2, 4, 9, 36),
+        ]
+        series = iris.pandas.as_series(cube)
+        self.assertArrayEqual(series, cube.data)
+        assert list(series.index) == expected_index
+
+    def test_time_360(self):
+        cube = Cube(np.array([0, 1, 2, 3, 4]), long_name="ts")
+        time_unit = cf_units.Unit(
+            "days since 2000-01-01 00:00", calendar=cf_units.CALENDAR_360_DAY
+        )
+        time_coord = DimCoord(
+            [0, 100.1, 200.2, 300.3, 400.4], long_name="time", units=time_unit
+        )
+        cube.add_dim_coord(time_coord, 0)
+        expected_index = [
+            cftime.Datetime360Day(2000, 1, 1, 0, 0),
+            cftime.Datetime360Day(2000, 4, 11, 2, 24),
+            cftime.Datetime360Day(2000, 7, 21, 4, 48),
+            cftime.Datetime360Day(2000, 11, 1, 7, 12),
+            cftime.Datetime360Day(2001, 2, 11, 9, 36),
+        ]
+
+        series = iris.pandas.as_series(cube)
+        self.assertArrayEqual(series, cube.data)
+        self.assertArrayEqual(series.index, expected_index)
+
+    def test_copy_true(self):
+        cube = Cube(np.array([0, 1, 2, 3, 4]), long_name="foo")
+        series = iris.pandas.as_series(cube)
+        series[0] = 99
+        assert cube.data[0] == 0
+
+    def test_copy_int32_false(self):
+        cube = Cube(np.array([0, 1, 2, 3, 4], dtype=np.int32), long_name="foo")
+        series = iris.pandas.as_series(cube, copy=False)
+        series[0] = 99
+        assert cube.data[0] == 99
+
+    def test_copy_int64_false(self):
+        cube = Cube(np.array([0, 1, 2, 3, 4], dtype=np.int32), long_name="foo")
```

Comment:
```suggestion
        cube = Cube(np.array([0, 1, 2, 3, 4], dtype=np.int64), long_name="foo")
```
Although you are restoring these tests, this test does look wrong and this looks like an easy fix so worth doing?

Reply:
4a934cf5c2b9077537724175201a2292b81e4e50